ios/ReactNativeUsabilla/ReactNativeUsabilla.m

```diff
@@ -10,7 +10,7 @@
 #import <React/RCTBridgeModule.h>
 #import <React/RCTEventEmitter.h>
 
-@interface RCT_EXTERN_MODULE(UsabillaInterface, NSObject)
+@interface RCT_EXTERN_MODULE(UsabillaBridge, NSObject)
```

Comment:
I understand the change. Why do we rename it to Bridge ?

Reply:
Because at js it looks for the native module called as UsabillaBridge